src/editor/components/modals/history-modal.jsx

```diff
@@ -54,15 +91,21 @@ class HistoryModalUnconnected extends React.Component {
 
 export function mapStateToProps(state) {
   const notebookHistory = state.notebookHistory || {};
-  const { errorGettingRevisionList, revisionListFetchStatus } = notebookHistory;
+  const {
+    errorGettingRevisionList,
+    revisionListFetchStatus,
+    revisionContent,
+    selectedRevisionId
+  } = notebookHistory;
 
   return {
     errorGettingRevisionList,
-    revisionListFetchStatus
+    revisionListFetchStatus,
+    canRevert: selectedRevisionId !== undefined && revisionContent
```

Comment:
We should also disable / hide this button if the user is on the latest revision.

Reply:
With migration of logic to revision-diff, behavior is corrected.